ProcessLib/HeatTransportBHE/HeatTransportBHEProcess.cpp

```diff
@@ -197,6 +199,44 @@ void HeatTransportBHEProcess::computeSecondaryVariableConcrete(
         getDOFTable(process_id), t, x, _coupled_solutions);
 }
 
+NumLib::IterationResult HeatTransportBHEProcess::postIterationConcreteProcess(
+    GlobalVector const& x)
+{
+    // if the process use python boundary conditon
+    if (_process_data.if_bhe_network_exist_python_bc == false)
+        return NumLib::IterationResult::SUCCESS;
+    else
+    {
+        // Here the task is to get the outflow temperature,
+        // transfer it to TESPy and to get inflow temperature,
+        // and determine whether it converges.
+        auto const Tout_nodes_id =
+            std::get<3>(_process_data.py_bc_object->dataframe_network);
+        const std::size_t n_bc_nodes = Tout_nodes_id.size();
+        for (std::size_t i = 0; i < n_bc_nodes; i++)
+        {
+            // read the T_out and store them in dataframe
+            std::get<2>(_process_data.py_bc_object->dataframe_network)[i] =
+                x[Tout_nodes_id[i]];
+        }
+        // Tout transfer to Python
+        auto const tespy_result =
+            _process_data.py_bc_object->tespyThermalSolver(
+                std::get<1>(_process_data.py_bc_object->dataframe_network),
+                std::get<2>(_process_data.py_bc_object->dataframe_network));
+        auto const cur_Tin = std::get<2>(tespy_result);
+
+        auto const if_convergence = std::get<1>(tespy_result);
+        if (if_convergence == true)
+            return NumLib::IterationResult::SUCCESS;
+        else
+            for (std::size_t i = 0; i < n_bc_nodes; i++)
+                std::get<1>(_process_data.py_bc_object->dataframe_network)[i] =
```

Comment:
Braces around (even single line) if/else/for branches/loops missing.

Reply:
fixed in commit 87cce675